### PR TITLE
Ocr schema

### PR DESCRIPTION
**Pull request by BenMacnaughton**
Please ensure you've followed the checklist and provide all the required information *before* requesting a review.

## Required Information

- [ ] Reference the issue this is meant to fix or describe it if none exists.
- [ ] Full description of changes made.

## Pull Request Checklist

- [ ] Coding standards are followed (e.g., [PEP8](https://pep8.org/))
- [ ] An entry exists for the container in `docker-compose.yml` or `build.yml`
- [ ] The Docker image builds
- [ ] The container runs correctly when sent inputs with the changes
- [ ] No models or other large files are part of these commits
- [ ] A description of the tests already run as part of this PR is present

### CODE REVIEW

**Review by aidanwilliams09 — APPROVED** [review]

File: preprocessors/ocr.schema.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                    "text": {
+                        "type": "string",
+                        "title": "Text",
+                        "description": "Text contained within the line"
+                    },
+                    "bounding_box": {
+                        "type": "array",
+                        "title": "Bounding box",
+                        "description": "The bounding box of the text in pixels",
+                        "minItems": 8,
```
Comment (JRegimbal) [inline preprocessors/ocr.schema.json:24]: What are the 8 items? Please specify in the description.

File: preprocessors/ocr.schema.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/preprocessors/ocr.schema.json",
+    "type": "object",
+    "title": "Autour Maps Preprocessor",
```
Comment (JRegimbal) [inline preprocessors/ocr.schema.json:5]: Wrong title

File: preprocessors/ocr.schema.json [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    "description": "Information on the contents of a place on a map",
```
Comment (JRegimbal) [inline preprocessors/ocr.schema.json:6]: Wrong description

**Review by JRegimbal — CHANGES_REQUESTED** [review]

File: preprocessors/ocr.schema.json (hunk 1 above)
Comment (BenMacnaughton) [reply preprocessors/ocr.schema.json:24]: Sorry about the ambiguity. Each has been specified

**Comment by BenMacnaughton** [comment]
Sorry for the ambiguity. I think the new description should be clear

**Review by JRegimbal — APPROVED** [review]

**Pull request merged**